dist/main.js.LICENSE.txt

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
- [ ] [OPTIONAL] if you're having difficulty thinking of writing a professional PR summary consider using the project requirements as an example then it will look something like below:
## Milestone 1: list structure
- Set up a new project with webpack.
- Created an **index.html** file 
- Set the **index.html** file as a template using the *HTML Webpack Plugin*.
- Created an **index.js** file and set an array of some simple *to do tasks*.
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- Created a **style.css** and set rules for the **To Do List**

Reply:
Thanks for this guide.